app/components/e-fieldset.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <fieldset :class="b()">
+    <legend v-if="legend" :class="b('legend')"><span :class="b('inner')">{{ legend }}</span></legend>
+    <div :class="b('fields')"><slot></slot></div>
+  </fieldset>
+</template>
+
+<script>
+
+  /**
+   *
+   * Component to render a fieldset with or without legend.
+   *
+   */
+  export default {
+    name: 'e-fieldset',
+    // components: {},
+    // mixins: [],
+
+    props: {
+      /**
+       * Defines the legend of the fieldset
+       *
+       */
+      legend: {
+        type: String,
+        default: null,
+      },
+    },
+    // data() {
+    //   return {};
+    // },
+
+    // computed: {},
+    // watch: {},
+
+    // beforeCreate() {},
+    // created() {},
+    // beforeMount() {},
+    // mounted() {},
+    // beforeUpdate() {},
+    // updated() {},
+    // activated() {},
+    // deactivated() {},
+    // beforeDestroy() {},
+    // destroyed() {},
+
+    // methods: {},
+    // render() {},
+  };
+</script>
+
+<style lang="scss">
+  .e-fieldset {
+    &__legend {
+      @include font($font-size--18, 22px, $font-weight--regular);
+
+      color: var(--theme-color-secondary-1);
+      margin-bottom: $spacing--25;
```

Comment:
- [ ] I would set the spacing to the `.e-fieldset__fields`, so the fieldset has a padding arround